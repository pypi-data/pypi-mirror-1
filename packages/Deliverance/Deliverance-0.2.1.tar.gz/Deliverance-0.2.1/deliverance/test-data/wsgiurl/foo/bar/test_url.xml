<rules xmlns="http://www.plone.org/deliverance">
</rules>
