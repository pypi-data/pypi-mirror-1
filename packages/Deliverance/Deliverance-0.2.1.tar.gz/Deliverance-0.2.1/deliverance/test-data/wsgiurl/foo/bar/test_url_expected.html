  <html>
     <head>
       <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
       <style type="text/css">
         @import url(http://localhost/foo/bar/whatever.css)
       </style>
     </head>
     <body>
        <a href="http://localhost/foo/bar/quux/zoop">zoop</a>
     </body>
  </html>
